owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewVideoActivity.kt

```diff
@@ -92,7 +96,7 @@ class PreviewVideoActivity : FileActivity(), Player.Listener, OnPrepareVideoPlay
     private var playWhenReady = true
     private var playbackPosition: Long = 0
     private var windowInsetsController: WindowInsetsControllerCompat? = null
-    private val previewVideoViewModel: PreviewVideoViewModel by viewModel()
+    private val previewVideoViewModel: PreviewVideoViewModel by viewModel { parametersOf(intent.getParcelableExtra(EXTRA_FILE)) }
```

Comment:
In any case, can't you extract the file from `arguments` in this case? Does it need to be from `intent`?

Reply:
It's not possible because it's an `Activity` and `requireArguments()` only exists for `Fragments`